modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductOptionValueResourceImpl.java

```diff
@@ -188,7 +194,7 @@ private ProductOptionValue _toProductOptionValue(
 		_cpDefinitionOptionValueRelService;
 
 	@Reference
-	private CPInstanceService _cpInstanceService;
+	private CPInstanceLocalService _cpInstanceLocalService;
```

Comment:
Use the remote service (to check if you have the permission to perform the action)